content/en/users/getting-started/cli/_index.md

```diff
@@ -48,6 +48,7 @@ FedCloud client has the following modules (features):
 The FedCloud client can be installed with the `pip3` Python package manager
 (without root or aministrator privileges).
 
+<!-- markdownlint-disable no-space-in-emphasis -->
```

Comment:
Instead of setting this exclusion it would be cleaner to use `$VIRTUAL_ENV/bin` instead of _$VIRTUAL_ENV/bin_.
It makes also things more distinguishable and I think it makes sense to use backticks for paths.

Reply:
Fixed